gnomad/utils/vcf.py

```diff
@@ -42,7 +42,7 @@
 Global populations that are included in filtering allele frequency (faf) calculations. Used in VCF export.
 """
 
-SEXES = ["male", "female"]
+SEXES = ["XX", "XY"]
 """
 Sample sexes used in VCF export.
```

Comment:
```suggestion
Sample sexes used in VCF export.

```
(not sure if there needs to be a space here)